src/Symfony/Bundle/DebugBundle/Resources/views/Profiler/dump.html.twig

```diff
@@ -39,42 +35,23 @@
 {% endblock %}
 
 {% block menu %}
-    <span class="label">
-        <span class="icon">
-        {{- "" -}}
-        <svg width="28" height="28" xmlns="http://www.w3.org/2000/svg" version="1.1" x="0px" y="0px" viewBox="0 0 28 28" enable-background="new 0 0 28 28" xml:space="preserve"><path fill="#3F3F3F" d="M28 13h-1.1C26.5 6.6 21.4 1.5 15 1.1V0h-2v1.1C6.6 1.5 1.5 6.6 1.1 13H0v2h1.1C1.5 21.4 6.6 26.5 13 26.9 V28h2v-1.1c6.4-0.5 11.5-5.6 11.9-11.9H28V13z M15 24.9V19h-2v5.9c-5.3-0.5-9.5-4.7-9.9-9.9H9v-2H3.1C3.5 7.7 7.7 3.5 13 3.1V9h2 V3.1c5.3 0.5 9.5 4.7 9.9 9.9H19v2h5.9C24.5 20.3 20.3 24.5 15 24.9z"/></svg>
-        {{- "" -}}
-        </span>
-        <strong>dump()</strong>
-        <span class="count">
-            <span>{{ collector.dumpsCount }}</span>
-        </span>
+    <span class="label {{ collector.dumpsCount == 0 ? 'disabled' }}">
+        <span class="icon">{{ include('@Debug/Profiler/icon.svg') }}</span>
+        <strong>Debug</strong>
     </span>
 {% endblock %}
 
 {% block panel %}
-    <h2>dump()</h2>
+    <h2>Dumped Contents</h2>
 
-    <style>
-        li.sf-dump {
-            list-style-type: disc;
-        }
-        .sf-dump ol>li {
-            padding: 0;
-        }
-        .sf-dump a {
-            cursor: pointer;
-        }
-        .sf-dump-compact {
-            display: none;
-        }
-    </style>
-
-    {% if collector.dumpsCount %}
-        <ul class="alt">
-            {% for dump in collector.getDumps('html') %}
-            <li class="sf-dump sf-reset">
-                in
+    {% if collector.dumpsCount == 0 %}
```

Comment:
this can be simplified using the ``{% for %} {% else %} {% endfor %}`` Twig construct (it was not possible previously because of the ``ul`` tag)